.github/workflows/build.yaml

```diff
@@ -85,47 +79,87 @@ jobs:
           # Build, tag and push the image
           ./gradlew jib
         env:
-          GCR_TAG: ${{ needs.bump_version.outputs.api_image_tag }}
+          GCR_TAG: ${{ needs.bump_version.outputs.tag }}
+
+  get-branch-name:
+    runs-on: ubuntu-latest
+    needs:
+      - bump_version
+    outputs:
+      branch-name: ${{ steps.determine-branch-name.outputs.branch-name }}
+    steps:
+      - name: Determine branch name
+        id: determine-branch-name
+        run: |
+          echo "Branch name: ${GITHUB_HEAD_REF}"
+          echo "branch-name=${GITHUB_HEAD_REF}" >> $GITHUB_OUTPUT
+        shell: bash
```

Comment:
I vaguely recall explicit `shell` being required for composite actions, but usually it's not needed. Is it needed here?

Reply:
Good point, probably not, since this is a very barebones script, bash shouldn't be needed. I'll remove it and if the step fails, I'll revert.